solver/src/liquidity/uniswap.rs

```diff
@@ -68,26 +70,33 @@ impl UniswapLiquidity {
 
             let future = pair_contract.get_reserves().batch_call(&mut batch);
             vacant.insert(future);
+        };
+
+        for order in offchain_orders {
+            get_reserves_if_not_yet_tracked(
+                TokenPair::new(order.buy_token, order.sell_token).expect("buy token = sell token"),
+            );
+
+            // Include every token with native token pair in the pools
+            TokenPair::new(order.buy_token, self.native_token_wrapper)
+                .map(&mut get_reserves_if_not_yet_tracked);
```

Comment:
I find it a little cleaner to use `if let` here instead of `map` because we do not actually want to map, we just call it for the side effect. This works for `Option` but for example for `Iterator` it wouldn't because iterators are lazy.